iphone/Maps/Core/iCloud/CloudStorageManager.swift

```diff
@@ -228,64 +255,74 @@ private extension CloudStorageManager {
   func writingResultHandler(for event: OutgoingEvent) -> WritingResultCompletionHandler {
     return { [weak self] result in
       guard let self else { return }
-      DispatchQueue.main.async {
-        switch result {
-        case .success:
-          // Mark that initial synchronization is finished.
-          if case .didFinishInitialSynchronization = event {
-            UserDefaults.standard.set(true, forKey: kUDDidFinishInitialCloudSynchronization)
-          }
-        case .reloadCategoriesAtURLs(let urls):
+      switch result {
+      case .success:
+        // Mark that initial synchronization is finished.
+        if case .didFinishInitialSynchronization = event {
+          UserDefaults.standard.set(true, forKey: kUDDidFinishInitialCloudSynchronization)
+        }
+      case .reloadCategoriesAtURLs(let urls):
+        DispatchQueue.main.async {
           urls.forEach { self.bookmarksManager.reloadCategory(atFilePath: $0.path) }
-        case .deleteCategoriesAtURLs(let urls):
+        }
+      case .deleteCategoriesAtURLs(let urls):
+        DispatchQueue.main.async {
           urls.forEach { self.bookmarksManager.deleteCategory(atFilePath: $0.path) }
-        case .failure(let error):
-          self.processError(error)
         }
+      case .failure(let error):
+        self.processError(error)
       }
     }
   }
 
   // MARK: - Error handling
   func processError(_ error: Error) {
-    if let synchronizationError = error as? SynchronizationError {
-      LOG(.debug, "Synchronization error: \(error.localizedDescription)")
-      switch synchronizationError {
-      case .fileUnavailable: break
-      case .fileNotUploadedDueToQuota: break
-      case .ubiquityServerNotAvailable: break
-      case .iCloudIsNotAvailable: fallthrough
-      case .failedToOpenLocalDirectoryFileDescriptor: fallthrough
-      case .failedToRetrieveLocalDirectoryContent: fallthrough
-      case .containerNotFound:
+    switch error {
+    case let syncError as SynchronizationError:
+      switch syncError {
+      case .fileUnavailable,
+           .fileNotUploadedDueToQuota,
+           .ubiquityServerNotAvailable:
+        LOG(.warning, "Synchronization Warning: \(syncError.localizedDescription)")
```

Comment:
Why localized? Will it be in French for French locale?

Reply:
Wait, users won't see this error, right? It's for us, to read in logs. It's better to have it in English, without localization.

Reply:
The errors messages that related to the uploading/downloading(when there is no connection) or quote errors will be visible to the users in the settings screen.